common/journal_entries/dvg_hre.txt

```diff
@@ -0,0 +1,121 @@
+﻿je_hre_internal = {
+	icon = "gfx/interface/icons/event_icons/event_portrait.dds"
+
+	complete = {
+		cu:north_german = {
+			NOT = { culture_is_discriminated_in = ROOT }
+		}
+		has_law = law_type:law_state_religion
+		has_law = law_type:law_racial_segregation
+		has_law = law_type:law_autocracy
+	}
+
+	on_complete = {
+		#trigger_event = {
+		#	id = hre.1
+		#}
+		custom_tooltip = HRE_success_tt
+		set_global_variable = hre_success_1
+	}
+
+	fail = {
+		has_revolution = yes
+	}
+
+	# 12 Years
+	timeout = 4380
+d
+
+	on_timeout = {
+		set_global_variable = hre_fail
+		#trigger_event = {
+			#id = elbian_confederation.4
+		#}
+	}
+
+	invalid = {
+		has_law = law_type:law_landed_voting
+		has_law = law_type:law_wealth_voting
+		has_law = law_type:law_total_separation
+	}
+
+	weight = 100
+	should_be_pinned_by_default = yes
+}	
+
+je_hre_external = {
+	icon = "gfx/interface/icons/event_icons/event_military.dds"
+
+	complete = {
+		country_or_subject_owns_entire_state_region = STATE_BOHEMIA
```

Comment:
Definitely needs a custom TT for the state ownership as well, something like "We own all of the states that make up the HRE" or something(it can be checked from formables menu might also be a good inclusion)